vernac/g_vernac.mlg

```diff
@@ -72,16 +72,25 @@ let parse_compat_version = let open Flags in function
     CErrors.user_err ~hdr:"get_compat_version"
       Pp.(str "Unknown compatibility version \"" ++ str s ++ str "\".")
 
+(* XXX: Locations here are messy, fix before merge. *)
```

Comment:
so not to forget it

Reply:
still there

Reply:
Yup, will improve once the rest of code (that is, use points) is known to be stable.